src/app/api/services/dot-content/dot-content.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { CoreWebService } from 'dotcms-js';
+import { Observable } from 'rxjs';
+import { RequestMethod } from '@angular/http';
+import { pluck, take } from 'rxjs/operators';
+
+@Injectable({
+    providedIn: 'root'
```

Comment:
Why?

Reply:
CLI stuff, to keep a service singleton. removed, think is not needed in our case.